commands/keplr.js

```diff
@@ -170,6 +167,38 @@ const keplr = {
 
     return true;
   },
+  async handleSelectChain() {
+    const chainNameExists = await playwright.waitForAndCheckElementExistence(
+      onboardingElements.chainNameSelector,
+    );
+
+    if (chainNameExists) {
+      await playwright.waitAndClickByText(
+        onboardingElements.chainName,
+        playwright.keplrWindow(),
+      );
+      await playwright.waitAndClick(
+        onboardingElements.submitChainButton,
+        playwright.keplrWindow(),
+      );
+      const importButtonExists =
+        await playwright.waitForAndCheckElementExistence(
+          onboardingElements.importButtonSelector,
+        );
+
+      if (importButtonExists) {
+        await playwright.waitAndClick(
+          onboardingElements.importButtonSelector,
+          playwright.keplrWindow(),
+        );
+      }
+    } else {
+      await playwright.waitAndClick(
+        onboardingElements.submitChainButton,
+        playwright.keplrWindow(),
+      );
+    }
+  },
```

Comment:
i think we can adapt this to handle chains dynamically i.e a user could provide us with an array ['Agoric', 'Cosmos hub', 'Secret'] and we could select all chains that match that name

Reply:
Agreed. We can add that in a separate PR or add to our TODOs.